### PR TITLE
Test data

### PR DESCRIPTION
**Pull request by omernu**
file for insert data to Jobs table
file for insert data to User table

### CODE REVIEW

**Review by Yarboa — COMMENTED** [review]
Omer i  do not understand the schema
Look you have the following on models.py

You have class User and class Company
Please extend existing model file with your additional staff.
- You need to work with ORMs it is class mapping for Database objects, OK?
- Please concentrate in filling the correct content with the updates classes
Once you run the following: pipenv run python manage.py makemigrations ultimatejobweb
It should create the updated schema with content, verify from admin url, that you have content in the tables with the correct data,
After that it will be easy to add some tests, 
 
 

-

File: search_engine.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import sqlite3
```
Comment (simzacks) [inline search_engine.py:1]: You should not have any direct interaction with the DB, it should only use the ORM.

File: ultimatejobweb/migrations/test_data_jobs.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+        test_data = [
+          ('1', 'omer', 'Facebook', 'Production Engineer',
+            'https://www.facebook.com/careers/jobs/1672813472870915/'),
+          ('2', 'rom', 'Facebook', 'Production Engineer',
+            'https://www.facebook.com/careers/jobs/712878282607325/'),
+        ]
+
+        with transaction.atomic():
+            for id, user, CompanyName, JobTitle, ApplyURL in test_data:
```
Comment (simzacks) [inline ultimatejobweb/migrations/test_data_jobs.py:21]: `u = User.objects.get(username=user)`
This gives you a User object.
Then you add `user=u` in the Jobs() constructor

File: search_engine.py (hunk 1 above)
Comment (omernu) [reply search_engine.py:1]: the search engine file is not relevant for now, idk why it's in this PR.

**Comment by omernu** [comment]
hi, i tried to add another file using git commit --amend, but it doesn't push it to the PR, and when i try to push this error shows up:  ! [rejected]        main -> test_data (non-fast-forward)
error: failed to push some refs to 'https://github.com/omernu/ultimatejob.git'
hint: Updates were rejected because a pushed branch tip is behind its remote
hint: counterpart. Check out this branch and integrate the remote changes
hint: (e.g. 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

**Pull request closed without merging**